guava/src/main/java/org/jdbi/v3/guava/GuavaMappers.java

```diff
@@ -51,7 +52,7 @@ public static ColumnMapperFactory columnFactory() {
 
             if (ImmutableList.class.equals(erasedType)) {
                 return GenericTypes.findGenericParameter(type, ImmutableList.class)
-                        .flatMap(config::findColumnMapperFor)
+                        .flatMap(t -> config.get(ColumnMappers.class).findFor(type, config))
```

Comment:
This might be splitting hairs, but it should probably be `findFor(t, config)` (`t` instead of `type`)